django/thunderstore/schema/sync.py

```diff
@@ -0,0 +1,84 @@
+import requests
+from django.db import transaction
+
+from thunderstore.community.models import (
+    Community,
+    PackageCategory,
+    PackageListingSection,
+)
+from thunderstore.core.settings import ECOSYSTEM_SCHEMA_ENDPOINT
+from thunderstore.schema.schema import Schema, SchemaGame
+
+
+# TODO: Add support for deleting or at least disabling unnecessary content
+@transaction.atomic
+def import_game_community(identifier: str, game: SchemaGame):
+    if not game.thunderstore:
+        return
+
+    config = game.thunderstore
+
+    if not (community := Community.objects.filter(identifier=identifier).first()):
```

Comment:
Assignment expressions are cool and all, but would Django's `get_or_create` or `update_or_create` be more idiomatic? This comment also applies for `PackageCategory` and `PackageListingSection` stuff below.

Reply:
The problem with those is that you might want to match the object with different criteria than you'd want to update or create it with. Maybe `update_or_create` is flexible enough to handle everything correctly, but I seem to remember that it wasn't.

That said, this area is a good spot for improvements later on. Really what we should use is an upsert query (bulk insert that on conflict updates specific fields instead), but Django doesn't have built in support for it and I didn't want to add needless complexity here yet.

Very valid suggestions though and I agree with the sentiment